PlannedToAT/Controllers/HomeController.cs

```diff
@@ -40,7 +40,7 @@ public IActionResult SignUpStudent()
         }
         public IActionResult StudentDashboard(string studentName, DateTime dob, string race, string phone, string email, string institution, string subgroup)
         {
-            var model = new StudentSignUpModel
+            var model = new SignUpStudent
```

Comment:
not sure how this worked as the file name and the class name were different and would not compile